vllm/model_executor/guided_decoding/outlines_logits_processors.py

```diff
@@ -61,7 +61,7 @@ def __call__(self, input_ids: List[int],
                           -math.inf,
                           device=scores.device)
         mask[allowed_tokens] = 0
-        scores.add_(mask)
+        scores = scores.add(mask)
```

Comment:
just curious, what's the difference here?

Reply:
should be no difference in result, however, 'add_' is executing in_place, while 'add' will return a tensor.
From my test, using 'add_' leads to "RuntimeError: synNodeCreateWithId failed for node: strided_insert with synStatus 1 [Invalid argument]. ", replacing with 'add' fixed above issue